app/build.gradle

```diff
@@ -1,7 +1,7 @@
 apply plugin: 'com.android.application'
 
 android {
-    compileSdkVersion 22
+    compileSdkVersion 23
```

Comment:
This change has been made to use the latest API for TabLayout which has been used later in **Safety Plan**
However, since targetSDK is the same, it doesn't affect anything